
builtin_interfaces/Time stamp

float64 servo_inner_tilt_angle_degrees
float64 servo_outer_tilt_angle_degrees
float64 servo_inner_pwm
float64 servo_outer_pwm

float64 motor_delta_pwm
float64 motor_average_pwm
float64 motor_upwards_pwm
float64 motor_downwards_pwm
